js/components/sidebar-menu.tsx

```diff
@@ -52,13 +52,15 @@ export default class SidebarMenu extends BaseComponent<IProps, IState> {
   toggleVelocities: any;
   toggleVolcanicEruptions: any;
   toggleWireframe: any;
+  toggleMetamorphism: any;
```

Comment:
Not that it matters much, but isn't the type here just `() => void` for the `toggle...` functions and then `(value: any) => void` for the `change...` functions?

Reply:
Thanks, I've added these types.